ext/xmlrpc/xmlrpc.stub.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+function xmlrpc_encode($value): string {}
+
+function xmlrpc_decode(string $xml, string $encoding = "iso-8859-1") {}
```

Comment:
We can also specify the `void` return type here, so:
```suggestion
function xmlrpc_decode(string $xml, string $encoding = "iso-8859-1"): void {}
```

Reply:
Test seems to disagree: https://github.com/php/php-src/blob/e3abc534eb98add83d9f705d864f1d0fdd79e5ea/ext/xmlrpc/tests/006.phpt#L22-L23

Reply:
Oh, you're right! `xmlrpc_decode` can actually return any type. Thanks for double-checking! :)